cs/index.json

```diff
@@ -99,6 +99,7 @@
     "wattage_cost": "Wattage cost",
     "mining_electricity_consumption": "Mining electricity consumption of your GPUs/CPUs",
     "electric_cost": "Electricity cost for 1 kWh",
-    "estimate_again": "Estimate again"
+    "estimate_again": "Estimate again",
+    "auto_update_wattage" : "Auto update from Miner tool"
```

Comment:
This key name doesn't really align with the text? What exactly are we trying to inform the user about?

Reply:
We have new checkbox on Wattage Setting called "Auto update from Miner tool" 
https://gitlab.com/mcnfreedom/vn-team/crypto.miner.client/uploads/38af60078d10d6c581a8fcbceb017ff8/image.png

That means `wattage` value will be updated automatically if Crypto Miner can get this info from Miner tool

Reply:
Hum okay, I was going to suggest the following, but I guess there's not much difference.
use_detected_wattage: "Use wattage data obtained from Miner"